pkg/etc/init.d/wavefront-proxy

```diff
@@ -102,7 +102,14 @@ jsvc=$proxy_dir/bin/jsvc
 
 jsvc_exec()
 {
+    if [[ ! $1 == "-stop" ]]; then
+        > $daemon_log_file
```

Comment:
Daemon log has always been overwritten, since it was a redirect of stdout anyway. These two logs have very limited use (we haven't had to use them yet), so I believe that keeping these logs just for the last run is sufficient - one less thing to maintain and rotate.